core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -1129,7 +1141,9 @@ public void testCompleteCreateTransaction() {
     Assert.assertEquals("Table properties should be a superset of the requested properties",
         properties.entrySet(),
         Sets.intersection(properties.entrySet(), table.properties().entrySet()));
-    Assert.assertEquals("Table location should match requested", "file:/tmp/ns/table", table.location());
+    if (!supportsServerManagedLocation()) {
```

Comment:
Should this be `supportsClientLocations`? I think it's reasonable to say that the server doesn't support client requested locations, rather than "supports" managing locations (which is to say it overrides them).

Or may be this should be `overridesRequestedLocation()`?

Reply:
I like `overridesRequestedLocation()`, that seems more clear. I'll make that change.

Reply:
This is done